app/models/section.rb

```diff
@@ -126,6 +126,7 @@ def publish!
 
   def withdraw_and_mark_as_exported!(exported_at = Time.zone.now)
     latest_edition.exported_at = exported_at
+    latest_edition.save!
```

Comment:
`latest_edition.archive` on the line below calls `save!` under the hood, so I think this is superfluous. Hopefully there aren't any section editions that are already withdrawn that don't have an `exported_at` attribute

Reply:
I changed one of the tests to stop using mocks and actually assert for exported_at and that test was starting to fail without this line added.